### ISSUE TITLE
Add Pijin and Tok Pisin to the list of possible form translations 

### ISSUE THREAD

**Issue by janagombitova** [akvo/akvo-flow#2202]
When a user wants to translate a form, she needs to select the language first, The list currently holds around 180 languages. 

Currently we don't have Solomon Islands Pijin as a language option for form translations. The closest is Bislama - which is used in Vanuatu, but is a different language. Pijin uses the same alphabet and English. 

Could we have Pijin added as an option, as well?

**Comment by janagombitova** [comment]
Created via Reamaze:

Link: https://flowhelp.reamaze.com/admin/conversations/akvoflow-support-request-number-65486-from-nikki-at-akvo-dot-org
Assignee: Jana Gombitova

Message:
[AKVOFLOW] Support request #65,486 from nikki@akvo.org      

User email: nikki@akvo.org  

Request Info
Device:  Personal computer
Browser:  Chrome 59.0.3071.115
OS:  OS X 10.12.5
IP Address:  203.110.204.4
URL:  http://flowsupport.akvo.org/form  

1- What can we help you with?
I would like to request a feature  

2- Who will be using this feature?
All enumerators working in Solomon Islands  

3- How do you want to use it?
To enable enumerators to conduct survey in their own language  

4- Why do you need to have it?
Currently languages available excludes their own language.  

Tell us more about your request
Currently we don't have Solomon Islands Pijin as a language option for survey translations. The closest is Bislama - which is used in Vanuatu, but is a different language. Pijin uses the same alphabet and English. Could we have Pijin added as an option, as well?

**Comment by janagombitova** [comment]
A second request submitted via support to add Birmese Zawgyi to the list.

**Comment by janagombitova** [comment]
Created via Reamaze:

Link: https://flowhelp.reamaze.com/admin/conversations/akvoflow-support-request-number-81083-from-lissy-at-akvo-dot-org

**Comment by janagombitova** [comment]
A few more language requests to add to the list: "We currently use Bislama for all Melanesia creoles, but this is becoming a problem when we have Bislama, and another creole needing to be added. Also as they are different (albeit closely related languages), it make sense to have them in the list. Can we please have Tok Pisin and Solomon Islands Pijin added to the languages list for survey translations? Thank you!" 

To summarise the requests wish to see the following languages in the dropdown when selecting into which language Salim translates the form. 
* [ ] Pijin
* [ ] Birmese Zawgyi
* [ ] Tok Pisin

**Comment by janagombitova** [comment]
Created via Reamaze:

Link: https://flowhelp.reamaze.com/admin/conversations/akvoflow-support-request-number-92528-from-nikki-at-akvo-dot-org

**Comment by stellanl** [comment]
Tok Pisin has an ISO 639-2 code: "tpi"
Zawgyi may be a popular font/text encoding rather than a language; apparently the Myanmar script part of Unicode is not compatible...
http://my.duniakitab.com/ThanLwinSoft/ThanLwinSoft/MyanmarUnicode/Intro.php
Solomon Islands Pijin has no official status (according to Wikipedia), the ISO 639-3 code is "pis".

**Comment by stellanl** [comment]
We already have at least one 3-letter code (quc), so adding Tok Pisin and Pijin should only require editing Dashboard/app/js/lib/controllers/languages.js.

**Comment by Maseline** [comment]
Test plan
Open a survey , select show Form Basics
Under Form Basics, select Manage Translations
Under form translation, select the drop down to choose a language, Pijin and Tok Pisin should be among the options of languages  to pick from.

**Comment by janagombitova** [comment]
@Maseline the implementation works well on UAT1. But there is one detail we did not handle and that is the app. @valllllll2000 Mary added two new languages to the list of possible form translations. I am testing it on UAT1 > _1.9.30 tests > 1.9.30 All questions (not monitoring) and even thought the translations are saved to the form, I do not see them in the app. Is there something you need to adjust on the apps end?

**Comment by valllllll2000** [comment]
After looking quickly I saw that Pijin and Tok Pisin are not listed here https://github.com/akvo/akvo-flow-mobile/blob/develop/database/src/main/res/values/languages.xml which is the app language list. Shall we add them? I would need to know the language code for those also

**Comment by Maseline** [comment]
@valllllll2000 the language codes are: 
Tok Pisin has an ISO 639-2 code: "tpi".
Solomon Islands Pijin has an ISO 639-3 code: "pis"
